src/Integrations/ActiveCampaign/ActiveCampaignClient.php

```diff
@@ -0,0 +1,675 @@
+<?php
+
+/**
+ * ActiveCampaign Client integration class.
+ *
+ * @package EightshiftForms\Integrations\ActiveCampaign
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Integrations\ActiveCampaign;
+
+use EightshiftForms\Hooks\Variables;
+use EightshiftForms\Integrations\ActiveCampaign\ActiveCampaignClientInterface;
+use EightshiftForms\Rest\ApiHelper;
+use EightshiftForms\Settings\SettingsHelper;
+
+/**
+ * ActiveCampaignClient integration class.
+ */
+class ActiveCampaignClient implements ActiveCampaignClientInterface
+{
+	/**
+	 * Use general helper trait.
+	 */
+	use SettingsHelper;
+
+	/**
+	 * Use API helper trait.
+	 */
+	use ApiHelper;
+
+	/**
+	 * Transient cache name for items.
+	 */
+	public const CACHE_ACTIVE_CAMPAIGN_ITEMS_TRANSIENT_NAME = 'es_active_campaign_items_cache';
+
+	/**
+	 * Transient cache name for item.
+	 */
+	public const CACHE_ACTIVE_CAMPAIGN_ITEM_TRANSIENT_NAME = 'es_active_campaign_item_cache';
+
+	/**
+	 * Return items.
+	 *
+	 * @param bool $hideUpdateTime Determin if update time will be in the output or not.
+	 *
+	 * @return array<string, mixed>
+	 */
+	public function getItems(bool $hideUpdateTime = true): array
+	{
+		$output = \get_transient(self::CACHE_ACTIVE_CAMPAIGN_ITEMS_TRANSIENT_NAME) ?: []; // phpcs:ignore WordPress.PHP.DisallowShortTernary.Found
+
+		// Check if form exists in cache.
+		if (empty($output)) {
+			$items = $this->getActiveCampaignLists();
+
+			if ($items) {
+				foreach ($items as $item) {
+					$id = $item['id'] ?? '';
+
+					$output[$id] = [
+						'id' => $id,
+						'title' => $item['name'] ?? '',
+					];
+				}
+
+				$output[ActiveCampaignClientInterface::TRANSIENT_STORED_TIME] = [
+					'id' => ActiveCampaignClientInterface::TRANSIENT_STORED_TIME,
+					'title' => \current_time('mysql'),
```

Comment:
As described here: https://make.wordpress.org/core/2019/09/23/date-time-improvements-wp-5-3/

It's better to use either `\wp_date(DATE_RFC3339, \time())`, or fetch the current date using the `\current_datetime()` which will return the instance of the `DateTimeImmutable` (which is, fun fact, btw not immutable 😂), then you can use something like `\current_datetime()->format('Y-m-d H:i:s')`.

Reply:
updated I used the `\current_datetime()->format('Y-m-d H:i:s')` here and all other instances